test-resources-hibernate-reactive/test-resources-hibernate-reactive-core/src/main/java/io/micronaut/testresources/hibernate/reactive/core/HibernateReactiveSupport.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2017-2021 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.testresources.hibernate.reactive.core;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * Provides constants and helper methods used by several test resources.
+ */
+public final class HibernateReactiveSupport {
+    static final String JPA = "jpa";
+    static final String DATASOURCES = "datasources";
```

Comment:
Hibernate Reactive should work without any `datasources` entries. If you want to test flyway+reactive or ordinary Hibernate + reactive, I recommend creating a special test project only for that purpose.

Reply:
It does work _without_ datasources, there are tests with, and without datasources. That's why we have the `-jdbc` and `-standalone` config files in tests.